offsetfetch.go

```diff
@@ -81,10 +81,25 @@ func (c *Client) OffsetFetch(ctx context.Context, req *OffsetFetchRequest) (*Off
 		})
 	}
 
-	m, err := c.roundTrip(ctx, req.Addr, &offsetfetch.Request{
-		GroupID: req.GroupID,
-		Topics:  topics,
-	})
+	var offsetFetchReq *offsetfetch.Request
```

Comment:
What about just changing topics to a nil slice?

e.g. change line 69 to

`var topics []offsetfetch.RequestTopic`